packages/amplify-category-api/src/__tests__/provider-utils/awscloudformation/ecs-apigw-stack.test.ts

```diff
@@ -59,6 +59,10 @@ describe('ecs stack', () => {
             },
           ],
         },
+        {
+          Effect: 'Allow',
+          Action: 's3:ListBucket',
```

Comment:
This will cause the test to throw an error without the change to `jsonPolicyToCdkPolicyStatement`.